katas/content/superposition/zero_and_bitstring/Verification.qs

```diff
@@ -0,0 +1,31 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Katas;
+
+    operation ZeroAndBitstringSuperposition_Reference (qs : Qubit[], bits : Bool[]) : Unit is Adj + Ctl {
+        H(qs[0]);
+
+        for i in 1 .. Length(qs) - 1 {
+            if bits[i] {
+                CNOT(qs[0], qs[i]);
+            }
+        }
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        let qubits = 3;
+        for i in 0 .. (2 ^ qubits) - 1 {
+            let bits = IntAsBoolArray(i, qubits);
+            Message($"Testing for bits = {bits}...");
+            if not CheckOperationsEquivalenceOnZeroStateWithFeedback(
+            Kata.ZeroAndBitstringSuperposition(_, bits),
+            ZeroAndBitstringSuperposition_Reference(_, bits),
+            qubits) {
+                return false;
+            }
+        }
+
+        true
```

Comment:
```suggestion
        return true;
```

Reply:
[Implicit return](https://github.com/microsoft/qsharp/wiki/Differences-from-the-previous-QDK#implicit-return) is new in modern QDK, so both syntaxes will work. It's a matter of personal preference and uniformity with other code in the kata :-)

Reply:
Nice! Good to know, thanks Mariia